plugins/modules/proxmox.py

```diff
@@ -226,8 +228,8 @@
   update:
     description:
       - If V(true), the container will be updated with new values.
+      - The current default value of V(false) is deprecated and should be replaced with V(update=true) in version 11.0.0.
```

Comment:
```suggestion
      - The current default value of V(false) is deprecated and will change to V(true) in community.general 11.0.0.
        Please set O(update) explicitly to V(false) or V(true) to avoid surprises and get rid of the deprecation warning.
```